src/main/java/edu/kit/ipd/crowdcontrol/objectservice/api/TemplateResource.java

```diff
@@ -0,0 +1,87 @@
+package edu.kit.ipd.crowdcontrol.objectservice.api;
+
+import edu.kit.ipd.crowdcontrol.objectservice.proto.Template;
+import edu.kit.ipd.crowdcontrol.objectservice.proto.TemplateList;
+import spark.Request;
+import spark.Response;
+
+/**
+ * Handles requests to template resources.
+ *
+ * @author Niklas Keller
+ */
+public class TemplateResource {
+    /**
+     * @param request
+     *         Request provided by Spark.
+     * @param response
+     *         Response provided by Spark.
+     *
+     * @return A list of all templates.
+     */
+    public TemplateList all(Request request, Response response) {
+        return TemplateList.newBuilder()
+                .addItems(Template.newBuilder().setId(1).setContent("{{TEST}}").build())
+                .addItems(Template.newBuilder().setId(2).setContent("{{TEST}}").build())
+                .addItems(Template.newBuilder().setId(3).setContent("{{TEST}}").build())
+                .build();
+    }
+
+    /**
+     * @param request
+     *         Request provided by Spark.
+     * @param response
+     *         Response provided by Spark.
+     *
+     * @return A single template.
+     */
+    public Template get(Request request, Response response) {
+        int id;
+
+        try {
+            id = Integer.parseInt(request.params(":id"));
+        } catch (NumberFormatException e) {
+            throw new BadRequestException(":id must be a valid integer.");
+        }
+
+        return Template.newBuilder().setId(id).setContent("{{TEST}}").build();
```

Comment:
Database operations do not exist yet. Well, partially for templates in 6e347052bbf0f5b1e6b653123eee24fe5c7d34ae.